### ISSUE TITLE
Need help trying to load 2 slides from 1 anchor

### ISSUE THREAD

**Issue by iontorres** [KevinBatdorf/liquidslider#169]
Hi, I've been trying to load 2 slides from different sliders containers and i need both to move when i make click to a single anchor, the thing is i just can't make this work is there a solution?

Thanks in advance :)

**Comment by KevinBatdorf** [comment]
I'm not 100% sure what you're trying to do, but each slider object has an API that you can control from anywhere. Check [this link](http://kevinbatdorf.github.io/liquidslider/examples/page1.html#advanced-controls) for some examples.

in psuedo code
```javascript
var api1 = $.data( $('#slider-1')[0], 'liquidSlider');
var api2 = $.data( $('#slider-2')[0], 'liquidSlider');
$(.yourButton).('click', '.yourContainer', function() {
  api1.setNextPanel(2);
  api2.setNextPanel(2);
});
```

**Comment by iontorres** [comment]
Yep, that helped a lot actually what i was trying to reach is when i have 2 sliders which need to be and load synchronized when I select an unique anchor called "Next". So the link and example in the download make great difference, Thanks a lot for the help.
